### PR TITLE
Add tool to generate summary of HyPhy analyses.

### PR DESCRIPTION
**Pull request by davebx**
This tool takes a set of HyPhy JSON output files and generates a summary
of site and segment annotations for that analysis.

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection

### CODE REVIEW

File: tools/hyphy/hyphy_summary.xml [hunk 1]
```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0"?>
+<tool id="hyphy_summary" version="1.0.0" name="HyPhy-Summary">
```
Comment (bgruening) [inline tools/hyphy/hyphy_summary.xml:2]: please add profile

File: tools/hyphy/hyphy_summary.xml [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+            <param name="cfel" value="summary/summary-in-cfel-2.json" />
+            <param name="bgm" value="summary/summary-in-bgm-2.json" />
+            <output name="summary" file="summary/summary-out-segment-2.json" compare="sim_size"/>
+            <output name="annotation" file="summary/summary-out-site-2.json" compare="sim_size"/>
+        </test>
+        <test>
+            <param name="mode" value="merge" />
+            <param name="summary_inputs" value="summary/summary-out-segment-1.json,summary/summary-out-segment-2.json" compare="sim_size" />
+            <param name="annotation_inputs" value="summary/summary-out-site-1.json,summary/summary-out-site-2.json" compare="sim_size" />
+            <output name="summary" file="summary/merge-out-segment-1.json" compare="sim_size"/>
```
Comment (bgruening) [inline tools/hyphy/hyphy_summary.xml:118]: sim_size is not a good comparison, can you assert some context?

**Review by bgruening — COMMENTED** [review]
* summary-in-fade-2.json
* merge-out-site-1.json

and a few others are also too large. Can you check that and reduce the size?

File: tools/hyphy/hyphy_summary.xml (hunk 2 above)
Comment (davebx) [reply tools/hyphy/hyphy_summary.xml]: It looks like these summaries are deterministic, so we _should_ be ok with the basic comparison.

**Comment by davebx** [comment]
@bgruening the largest file is now 600KB, that should be a lot more manageable.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks @davebx

**Pull request merged**